content-src/components/ActivityFeed/ActivityFeed.js

```diff
@@ -24,9 +24,7 @@ const CALENDAR_HEADINGS = {
 
 const ActivityFeedItem = React.createClass({
   getInitialState() {
-    return {
-      showContextMenu: false
-    };
+    return {showContextMenu: false};
```

Comment:
Personally I prefer the older syntax for readability, but not sure I grasp all the options on that ESLint page. I'll defer to anybody else with an opinion.